packages/fast-application-utilities/src/localization.ts

```diff
@@ -1,16 +1,23 @@
 /**
  * Expose ltr and rtl strings
+ * @deprecated - use @microsoft/fast-web-utilties instead
  */
 export enum Direction {
     ltr = "ltr",
     rtl = "rtl",
 }
 
+/**
```

Comment:
Are we migrating these in a separate PR?

Reply:
Yeah - I have a seperate PR to add to fast-web-utilities.

Reply:
I feel like this is related and could be in one PR, otherwise we have a moment in time where we can't publish, plus it would bump both of them as it's a feature for both packages.

Reply:
@chrisdholt then in that case I would say this PR is premature, the feature in the other package that allows for the deprecation needs to go in before this PR can be considered.

Reply:
@janechu agree - that's why the PR was pushed prior and is up for review prior to this. Feel free to hold until that goes in if you like.